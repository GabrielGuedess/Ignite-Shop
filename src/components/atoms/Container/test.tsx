import { render } from '@testing-library/react';

import { Container } from '.';

describe('<Container />', () => {
  it('should render the container correctly', () => {
    const { container } = render(<Container />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
