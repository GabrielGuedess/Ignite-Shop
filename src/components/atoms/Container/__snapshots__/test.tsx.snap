// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render the container correctly 1`] = `
<div
  class="c-jrnCVl"
/>
`;
